tests/Model/Todo/TodoTest.php

```diff
@@ -179,4 +183,91 @@ public function it_throws_an_exception_if_todo_is_closed(Todo $todo)
             TodoDeadline::fromString('2047-12-11 12:00:00')
         );
     }
+
+    /**
+     * @test
+     * @return Todo
+     */
+    public function it_adds_a_reminder_to_todo()
+    {
+        $todoId = TodoId::generate();
+        $userId = UserId::generate();
+        $reminder = TodoReminder::fromString('2047-12-31 12:00:00');
+        $todo = Todo::post('Do something tomorrow', $userId, $todoId);
+        $this->popRecordedEvent($todo);
+
+        $this->assertNull($todo->reminder());
+
+        $todo->addReminder($userId, $reminder);
+        $events = $this->popRecordedEvent($todo);
+
+        $this->assertCount(1, $events);
+
+        $this->assertInstanceOf(ReminderWasAddedToTodo::class, $events[0]);
+
+        $expectedPayload = [
+            'todo_id' => $todoId->toString(),
+            'user_id' => $userId->toString(),
+            'reminder' => $reminder->toString(),
+        ];
+
+        $this->assertEquals($expectedPayload, $events[0]->payload());
+
+        return $todo;
+    }
+
+    /**
+     * @test
+     * @param Todo $todo
+     * @depends it_adds_a_reminder_to_todo
+     * @return Todo
+     */
+    public function it_can_add_another_reminder_if_desired(Todo $todo)
+    {
+        $todo->addReminder($todo->assigneeId(), TodoReminder::fromString('2047-12-11 12:00:00'));
+        $events = $this->popRecordedEvent($todo);
+
+        $this->assertCount(1, $events);
+        $this->assertInstanceOf(ReminderWasAddedToTodo::class, $events[0]);
+
+        return $todo;
+    }
+
+    /**
+     * @test
+     * @param Todo $todo
+     * @depends it_adds_a_reminder_to_todo
+     */
+    public function it_throws_an_exception_if_reminder_is_in_the_past(Todo $todo)
+    {
+        $this->expectException(InvalidReminder::class);
```

Comment:
Travis is failing in php 5.5 because of this.
I was just following: https://thephp.cc/news/2016/02/questioning-phpunit-best-practices
Is php 5.5. relevant anyway for this repository. It uses docker/vagrant with php 5.6 by default